test/integration_tests/long/test_similarity.py

```diff
@@ -411,6 +411,24 @@ def test_end_to_end_index_scan_should_work_correctly_on_image_dataset_qdrant(sel
         # Cleanup
         self.evadb.catalog().drop_index_catalog_entry("testQdrantIndexImageDataset")
 
+    @chromadb_skip_marker
+    def test_end_to_end_index_scan_should_work_correctly_on_image_dataset_chromadb(
```

Comment:
Please update the circle ci or dev configuration, so the test will not be skipped.

Reply:
My bad, fixed the circleci config